packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -475,6 +481,45 @@ async function handler(
     ) {
       throw new HttpError({ statusCode: 403, message: ErrorCode.CancelledBookingsCannotBeRescheduled });
     }
+    const userReschedulingIsOwner = isUserReschedulingOwner(userId, originalRescheduledBooking?.user?.id);
+    let isTeamOwnerOrAdmin = false;
+    if (isTeamEventType) {
```

Comment:
We don't want to restrict this for team events or owner user only. Any normal user should also be able to check slots available for attendee.

Reply:
It is not for checking slots available for attendees. It is meant to make sure that only organiser (in case of an individual event-type) or a team owner/admin (in case of a team event) is able to reschedule the bookings.